ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2616,3 +2616,56 @@ periodics:
       resources:
         requests:
           cpu: 500m
+- agent: kubernetes
+  cluster: app.ci
```

Comment:
This needs to run on `multi01` cluster.

Reply:
`registry.ci.openshift.org/ocp/cli-jq:latest` is amd64 only. Should we add a node selector?

Reply:
Yes the node selector would be required as well.